BUILD.md

```diff
@@ -9,79 +9,165 @@ However this will result in an error message because the build.groovy script has
 
 
 Example:
-```
-$DBB_HOME/bin/groovyz build.groovy --workspace /u/build/repos --application app1 --outDir /u/build/out --hlq BUILD.APP1
+```sh
```

Comment:
Should 'sh' come before the triple ''' quotes?

i.e. 
sh '''
$DBB_HOME/bin/grooovyz ...
'''

Reply:
The triple quotes followed by sh or any script langauge just tells mark-down about the formatting of the snippet.